docs/deployment-process/issue-tracking/azure-devops.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Azure DevOps Work Item Tracking Integration
+description: Configure Azure DevOps work item tracking with Octopus.
+position: 40
+---
+
+**Octopus 2019.7.3** introduced support to integrate Octopus with Azure DevOps work items. The integration includes the ability to:
+- Automatically add links to Azure DevOps work items from releases and deployments in Octopus.
+- Retrieve release notes from Azure DevOps work item comments to assist in automating release note generation.
```

Comment:
Suggesting this edit to make sure it doesn't alter the meaning.
```suggestion
- Retrieve release notes from Azure DevOps work item comments for automatic release note generation.
```

Reply:
Ok done, and also applied to GitHub page.